src/routers/handlers/confirm-company/confirmCompany.ts

```diff
@@ -38,6 +38,7 @@ export class ConfirmCompanyHandler extends GenericHandler<ConfirmCompanyViewData
             backURL: companyLookup,
             company,
             address,
+            title: "Confirm this is the correct company - Provide identity verification details for a PSC or relevant legal entity",
```

Comment:
Why is this added here?  if there is to be a change to the page title shouldn't it be in the locales? `confirm_company_main_title`

Reply:
This title value is not used in the template, it retrieves it properly from locales.  I think it was added here just for the unit test, so I've removed that from the test checks